docs/operations/libraries/workflow_alpha_release.md

```diff
@@ -0,0 +1,21 @@
+## Alpha Release Workflow
+This describes the process for pushing an alpha release for a `library` module.
+A `library` module is a module or repository that is a dependency of one or more
+Neon `services` or `plugins`.
+
+### Application
+This process applies when making any change to a `library` module.
+
+### Process
+- Create a branch from the `dev` branch of the module.
+- Complete any changes, including test coverage, documentation updates, and
+  dependency changes.
+- Create a Pull Request to the `dev` branch.
+- If any tests fail, mark the Pull Request as a Draft and make necessary changes.
+- After tests are passing, request a review from a maintainer.
```

Comment:
Is this intended for internal use only? If you don't have the right privileges in the repo, you can't request a review. Also, worth noting that some contributors may struggle with the tests and need some help getting them passing.

Reply:
Ah, I forgot you can't request a review by default.. Any idea how other orgs deal with this? Just up to the maintainers to monitor and self-assign?

Reply:
I think so, or maybe there is some automation around labels that auto-assign reviewers? In Home Assistant, they assign based on the contributor of a given feature, or directly to the general maintainer group

Reply:
I think the docs cover it for now; when we figure out a process for assigning unassigned issues we can add a note here